CHANGES

```diff
@@ -152,6 +156,9 @@ Bug Fixes
   * finagle-thrift: Properly locate sub-classed MethodIface services to instantiate for serving
     BaseServiceIface implemented thrift services. ``RB_ID=907608``
 
+  * finagle-redis: The SentinelClient will no longer throw an NoSuchElementException when
```

Comment:
can you also add a note that this is a breaking API change?  we've renamed pendingCommands to linkPendingCommands.